libcxx/docs/ReleaseNotes/18.rst

```diff
@@ -109,6 +95,13 @@ LLVM 18
 LLVM 19
 ~~~~~~~
 
+- The ``LIBCXX_ENABLE_ASSERTIONS`` CMake variable that was used to enable the safe mode will be deprecated and setting
+  it will trigger an error; use the ``LIBCXX_HARDENING_MODE`` variable with the value ``safe`` instead. Similarly, the
+  ``_LIBCPP_ENABLE_ASSERTIONS`` macro will be deprecated (setting it to ``1`` still enables the safe mode in this
+  release while also issuing a deprecation warning). ``_LIBCPP_ENABLE_ASSERTIONS`` will be removed entirely in the next
+  release and setting it will become an error. See :ref:`the hardening documentation <using-hardening-modes>` for more
```

Comment:
Instead of saying 
```
``_LIBCPP_ENABLE_ASSERTIONS`` will be removed entirely in the next release
```

we could say

```
``_LIBCPP_ENABLE_ASSERTIONS`` will be removed entirely in LLVM 20
```

Or at that point we could remove that sentence altogether since it is contained in LLVM 20 upcoming deprecations.